### PR TITLE
Improved menu scrolling speed

### PR DESCRIPTION
**Pull request by QuadrupleTurbo**
The idea came from a pull request for vMenu a long time ago and I thought that this menu could definitely use it.

### CODE REVIEW

**Comment by manups4e** [comment]
🤔 did you test it? As being the scaleform awaited after each control pressed, there's already a delay

**Comment by QuadrupleTurbo** [comment]
I've tested it yes, it works lovely just like I remember from MenuAPI :P

**Comment by QuadrupleTurbo** [comment]
but yes, feel free to test and get back to me, I'd love to see happen

**Comment by manups4e** [comment]
Merged and improved in next Commit

File: ScaleformUI_Csharp/UIMenu.cs [hunk 1]
```diff
@@ -1969,28 +1976,78 @@ public async void ProcessControl(Keys key = Keys.None)
...
+                }
             }
 
             else if (HasControlJustBeenPressed(MenuControls.Select, key) && UpdateOnscreenKeyboard() != 0 && !IsWarningMessageActive())
             {
                 Select(true);
             }
+
+            // IsControlBeingPressed doesn't run every frame so I had to use this
+            if (!Game.IsControlPressed(0, Control.FrontendUp) && !Game.IsControlPressed(0, Control.FrontendDown) && !Game.IsControlPressed(0, Control.FrontendLeft) && !Game.IsControlPressed(0, Control.FrontendRight))
```
Comment (manups4e) [inline ScaleformUI_Csharp/UIMenu.cs:2019]: also.. for reference.. you could have used HasControlJustBeenReleased in the ScaleformUI API itself for this.
Check here https://github.com/manups4e/ScaleformUI/blob/master/ScaleformUI_Csharp/UIMenu.cs#L2000

**Pull request closed without merging**